devito/parameters.py

```diff
@@ -258,6 +258,29 @@ def wrapper(*args, **kwargs):
         return wrapper
 
 
+class switch_log_level(object):
```

Comment:
doesn't this belong to `logger.py`?

Reply:
moved, but inserted `import configuration` in in `__init__` to avoid circular imports